### ISSUE TITLE
Test the Dev Readme

### ISSUE THREAD

**Issue by VictoriaODell** [open-austin/influence-texas#46]
Help us make sure that there are no bugs in the readme and you can get started okay! 
https://github.com/open-austin/influence-texas/blob/master/README_DEV.rst

**Comment by VictoriaODell** [comment]
@sheadscott is looking into this right now! Overall consensus is that it's good and can get the code running but a few more instructions can help some beginners get startd.

**Comment by SheepGotoHeaven** [comment]
Hi all. After updating all my VBox/Vagrant/Ansible to the latest versions, I was able to get through the setup process just fine. Three comments:

1)  In the Vagrant section toward the bottom of the file, some of the bullet points and external links to Vagrant etc. weren't rendering correctly on Github. They've been fixed in commit `a2c14a5` and this issue referenced in a comment. I pushed that directly to master, since it was a non-opinionated readme fix. Hope that's alright.

2) I was having the same `ALLOWED_HOSTS` (can't serve "localhost") error that @jcoene seems to have run into. I tried adding `DJANGO_ALLOWED_HOSTS` to `env.sh` and re-provisioning, but that didn't seem to work for me. Eventually I just edited my `/etc/hosts` to include a mapping from one of the accepted domains to `127.0.0.1`. Will check out what @jcoene did one night soon and try again.

3) It might be nice to have a section at the top of the dev readme with system requirements. I know they're right there in the Vagrantfile, but I didn't check it carefully before building the first time and 2GB basically puts me in disk-thrashing territory. Just a thought ¯\\_(ツ)_/¯
